webui/react/src/components/kit/Input.tsx

```diff
@@ -49,6 +50,7 @@ interface TextAreaProps {
 }
 
 interface PasswordProps {
+  autofocus?: boolean;
```

Comment:
does passing `autoFocus` straight through like we do with the regular input not work?

Reply:
I just tried this locally on the password input and it works -- can you double check? Note that while the html attribute isn't camelcased, the antd atrribute is.